packages/react-ui/src/icons/Cross.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
we don't use react-feather anymore ?

Reply:
I was supposed to start a new ui from scratch and they were no react-feather icons in the new design, I had all the SVG provided.
But we won't be doing the new design so I guess there is no need to spend to much time on this review.
Still you can use this branch to potentially play with rebass. You might be more successful implementing it in a fashion you like !

Reply:
Ok, is it possible to extract the eslint addition and fix you do ?

Reply:
Absolutely !

Reply:
I'll let the big PR be merged first the I'll take it.